std_msgs/Point3DFloat32 center
std_msgs/Point3DFloat32 size
int32 max_depth
byte[] structure_data
